js/actions/types.js

```diff
@@ -25,20 +34,22 @@ export type Action =
     }
   | { type: 'UPDATE_USER_PROFILE_REQUEST_ACTION' }
   | { type: 'UPDATE_USER_PROFILE_ERROR_ACTION', data: { error: string } }
+  | ApplicationStateAction
```

Comment:
for consistency give type names to all actions

Reply:
This is a refacting i am already doing in the "migrate to redux-sagas" stories.

Reply:
👍